src/main/java/Duke.java

```diff
@@ -39,21 +32,29 @@ else if (userInput.startsWith("event")){
                 count = addEventTask(userInput, tasks, count);
             }
             else {
```

Comment:
It would be clearer if you can consider to wrap these if-else statements into "switch" cases